pontoon/base/migrations/0032_user_profile_visibility.py

```diff
@@ -28,7 +28,7 @@ class Migration(migrations.Migration):
             name="visibility_email",
             field=models.CharField(
                 choices=[
-                    ("Logged in users", "Logged in users"),
+                    ("Logged-in users", "Logged-in users"),
```

Comment:
This is a Django [migrations](https://docs.djangoproject.com/en/4.2/topics/migrations/) file, and these are not meant to be edited. They keep history of all changes made to the database model and allow you to restore any past version of it when you're moving back and for among the code history.

Please restore changes made to this file.

That also means any change to the database model requires you to create a new migrations file. Special commands are available to automatically create such files.

You've [changed](https://github.com/mozilla/pontoon/pull/2842/files#diff-4ebc5e9888b2962aecd470dd76f7d410b4e246f2d8323df57ed7dd0c8d5e3ade) the data model and to create a corresponding migration file, please run the following commands:

```bash
make shell
./manage.py makemigrations
./manage.py migrate
(press Ctrl + D to go back to the default prompt)
```

Once you're done, please commit the new file and the restored changes in the edited migration file 0032 and push them to the branch.